README.md

```diff
@@ -196,6 +195,15 @@ Now you can connect to the mysql running in the docker container, also to connec
 10. Click the green "Run" button or "Shift + F10" to start the application
 11. After starting the Tomcat server on port `8080`, attempt to access the dummy route `http://localhost:8080/test` using the `GET` method in Postman or ThunderClient while providing the `bearer token`. If the terminal displays `test123`, it indicates that the setup has been successful.
 
+## To Authenticate Yourself
+
+Using [Website - backend](https://github.com/Real-Dev-Squad/website-backend)
+On Local - https://localhost:3000/auth/github/login?redirectURL=https://staging-skilltree.realdevsquad.com/tasks?v2=true
```

Comment:
Just wanted to understand shouldn't the redirect URL on local also be `localhost:3000/tasks?v2=true`

Reply:
no redirect URL need not be of the localhost only

Reply:
Yeah it would still work, but to keep consistency with the env, would we not want it to be localhost itself? Can you help me understand if there is any reason to keep it staging? Just wondering as it confused me.  If it will work with localhost, can we change it for consistency? Shouldnt use our staging env for the redirect if we are on local right? I

Reply:
Discussed offline. TLDR : This is just a frontend redirection URL (could be anything else as well so no concerns there) and V2 param is used by the website backend to set the cookie.